EnglishFilter/sections/antiadblock.txt

```diff
@@ -550,6 +550,9 @@ uaclips.com,kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/76391
+mrexcel.com#%#//scriptlet("set-constant", "adblockDetector", "noopFunc")
+@@||mrexcel.com/img/sgw/adLoader.min.js$domain=mrexcel.com
```

Comment:
Move these rules to the Annoyances - `antiadblock.txt` without task comment.
`@@||mrexcel.com/img/sgw/adLoader.min.js$domain=mrexcel.com` this requires a special section. Move it under
```
! Common safari_cb_affinity(general)
!#if (adguard_ext_safari || adguard_app_ios || adguard_ext_android_cb)
!#safari_cb_affinity(general)
```